src/runtime/text.h

```diff
@@ -157,3 +157,18 @@ static inline const u8 *utf8_find_r(const u8 *x, character c)
     }
     return found;
 }
+
+static inline const u8 *utf8_findn_r(const u8 *x, bytes n, character c)
```

Comment:
It would be nice to just get rid of the unbounded version of this. PATH_MAX could be used as a limit in absence of an actual string length.

Reply:
OK, I split this PR into 2 commits, and the first commits gets rid of the unbounded function.